cylc/rose/utilities.py

```diff
@@ -442,3 +439,25 @@ def simplify_opts_strings(opts):
             seen_once.append(item)
 
     return ' '.join(reversed(seen_once))
+
+
+def dump_rose_log(dest_root, node):
+    """Dump a config node to a timestamped file in the ``log`` sub-directory.
+
+    Args:
+        dest_root (pathlib.Path):
+            Installed location of a flow.
+        node (Rose Config node):
+            Node to be dumped to file.
+
+    Returns:
+        String filepath of the dump file relative to the install directory.
+    """
+    dumper = ConfigDumper()
+    timestamp = DateTimeOperator().process_time_point_str(
+        print_format='%Y%m%dT%H%M%S%Z'
+    )
+    fpath = dest_root / f'log/conf/{timestamp}-rose-suite.conf'
+    fpath.parent.mkdir(exist_ok=True, parents=True)
+    dumper.dump(node, str(fpath))
+    return f'log/conf/{timestamp}-rose-suite.conf'
```

Comment:
Sorry to be picky, but I would use a var here, otherwise if this gets broken in future, it might be a bit hard to debug
```suggestion
    rel_path = f'log/conf/{timestamp}-rose-suite.conf'
    fpath = dest_root / rel_path
    fpath.parent.mkdir(exist_ok=True, parents=True)
    dumper.dump(node, str(fpath))
    return rel_path
```